robot.py

```diff
@@ -21,7 +21,7 @@ class Robot(magicbot.MagicRobot):
     turret_component: Turret
 
     # Controllers
-    shooter_controller: ShooterController
+    # shooter_controller: ShooterController
```

Comment:
This should not be commented out.

Reply:
I presume this was commented out because main is broken: https://github.com/thedropbears/pycubehammer/actions/runs/4540001581/jobs/8000433075#step:5:476

If this is a deal breaker as is, feel free to fix?

Reply:
what @auscompgeek mentioned is why I commented it out. Is there any harm in leaving it like this until we get to implementing the shooter controller? I guess the alternative is to define som thing as first and then all should be hunky dory

Reply:
I'll fix it upstream now